src/i18n/messages/cs-CZ.json

```diff
@@ -193,6 +193,7 @@
   "bRgcSJ": "Měnit pozadí mezi zprávami v chatu pro lepší čitelnost",
   "bUW+Jz": "Změní '<'zpráva smazána'>' zpět na původní zprávu uživatele a zvýrazní ji.",
   "bfaAsn": "Smaže určitá slova, věty, nebo uživatele z vašeho chatu.",
+  "c+VRfV": "Doporučené kanály",
```

Comment:
please undo these translation changes. these should only be synced with crowdin.

Reply:
Do you mean I should not edit the i18n messages at all, and so, remove the French translations ? 
Or, should I just remove this specific change in each file ?

Reply:
Don't update the translation files at all, anything in i18n should be untouched.

Reply:
Translation changes are reverted.